pkgs/development/tools/click/default.nix

```
@@ -0,0 +1,92 @@
+{ lib
+, fetchFromGitLab
+, buildPythonPackage
```

You're building a Python application that accesses its package, not just a bare importable package. Wrong set of magic.
```suggestion
, buildPythonApplication
```